### DISCUSSION TITLE
International Weather?

### DISCUSSION THREAD

**Discussion by head-dunce** [weather-gov/api#412] [Q&A]
I'm currently using the API's from https://stormglass.io for my work-in-progress site http://www.weatherpirate.com
Their API says it's getting weather info from NOAA so I'm trying to get it directly from your API. But when I try and use the coordinates here in the Turks and Caicos, I get an error. It seems to work fine if I use an address in the US. 
Can anyone point me in the right direction?
`{
    "correlationId": "137012b5",
    "title": "Data Unavailable For Requested Point",
    "type": "https://api.weather.gov/problems/InvalidPoint",
    "status": 404,
    "detail": "Unable to provide data for requested point 21.8386,-71.8115",
    "instance": "https://api.weather.gov/requests/137012b5"
}`

**Comment by scadergit** [comment]
It is likely only one of several sources, as NOAA/NWS only provides weather data for US and its territories.

**Comment by kollymars** [comment]
This could also be from aviationweather.gov.  They have international days as well.
